src/util-inl.h

```diff
@@ -341,6 +341,14 @@ bool StringEqualNoCaseN(const char* a, const char* b, size_t length) {
   return true;
 }
 
+inline uint32_t Add32Clamp(uint32_t x, uint32_t y) {
+  return x + y >= x ? x + y : static_cast<uint32_t>(-1);
+}
+
+inline uint64_t Add64Clamp(uint64_t x, uint64_t y) {
+  return x + y >= x ? x + y : static_cast<uint64_t>(-1);
+}
```

Comment:
This function seems unused.

Reply:
I added it for parity (I had plans to use it for something else) but I can remove it.

Reply:
I would prefer not to add functions that have no functionality.